File: Curso Logica de Programação Neri Neitzke/Algoritmos em Java/abc_crescente.java
// Prof Neri Aldoir Neitzke   ULBRA
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
/*Problema: Problema: Ler 3 valores INTEIROS para as variáveis A, B, C. Apresentar os valores dispostos em
ordem crescente. */
import javax.swing.JOptionPane;
class abc_crescente
{
   public static void main(String args[])
   {
        int a,b,c;
        a = Integer.parseInt(JOptionPane.showInputDialog("Digite o valor para A: "));
        b = Integer.parseInt(JOptionPane.showInputDialog("Digite o valor para B: "));
        c = Integer.parseInt(JOptionPane.showInputDialog("Digite o valor para C: "));
        if ((a <= b) && (a <= a))
          if (b <= c)
               JOptionPane.showMessageDialog(null,"Em ordem crescente ficou.: "+a+"-"+b+"-"+c);
          else
               JOptionPane.showMessageDialog(null,"Em ordem crescente ficou.: "+a+"-"+c+"-"+b);
        else  if ((b <= a) && (b <= c))
          if (a <= c)
               JOptionPane.showMessageDialog(null,"Em ordem crescente ficou.: "+b+"-"+a+"-"+c);
          else
               JOptionPane.showMessageDialog(null,"Em ordem crescente ficou.: "+b+"-"+c+"-"+a);
        else  if (a <= b)
               JOptionPane.showMessageDialog(null,"Em ordem crescente ficou.: "+c+"-"+a+"-"+b);
          else
               JOptionPane.showMessageDialog(null,"Em ordem crescente ficou.: "+c+"-"+b+"-"+a);


   }
}

File: Curso Logica de Programação Neri Neitzke/Algoritmos em Pascal/pascal/abc_crescente.pas
// Prof Neri Aldoir Neitzke         - ULBRA
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
{Problema: Ler 3 valores INTEIROS para as variáveis A, B, C. Apresentar os valores dispostos em ordem crescente. }
program abc_crescente;
var a, b, c : integer;
begin
    write('Digite o valor de A.: ');
    readln(a);//
    write('Digite o valor de B.: ');
    readln(b);//
    write('Digite o valor de C.: ');
    readln(c);//
    if (a <= b) and (a <= c) then
        if (B <= C) then
               writeln('Em ordem crescente ficou.: ',A,'-',B,'-',C)
          else
               writeln('Em ordem crescente ficou.: ',A,'-',C,'-',B)
    else  if (b <= a) and (b <= c) then
        if (a <= C) then
               writeln('Em ordem crescente ficou.: ',b,'-',a,'-',C)
          else
               writeln('Em ordem crescente ficou.: ',b,'-',C,'-',a)
    else if (a <= b) then
               writeln('Em ordem crescente ficou.: ',C,'-',a,'-',b)
          else
               writeln('Em ordem crescente ficou.: ',C,'-',b,'-',a);
    writeln('..................Final do Sistema..................... ');
    readln();
end.
